src/test/groovy/com/twcable/grabbit/cli/GrabbitCliSpec.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2014-2016 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twcable.grabbit.cli
+
+import spock.lang.Ignore
+import spock.lang.Specification
+
+class GrabbitCliSpec extends Specification {
```

Comment:
Are we getting more love here? I realize, lots of private, static stuff - but hopefully we can alleviate that a bit with some good modeling?